2-WebApp-graph-user/2-3-Multi-Tenant/Startup.cs

```diff
@@ -106,7 +85,7 @@ public void ConfigureServices(IServiceCollection services)
 
                         return Task.FromResult(0);
                     };
-                });
+                }, option => { });
```

Comment:
the second options callback should probably do the binding as well? `Configuration.Bind("AzureAD", options);` ?